docs/getting_started/features/fares.md

```diff
@@ -0,0 +1,304 @@
+
+| Files included                   | Fields included   |
+|----------------------------------|-------------------|
+|[fare_products.txt](/schedule/reference/#fare_productstxt)|`fare_product_id`, `fare_product_name`, `amount`, `currency`, `fare_media_id` |
+|[fare_leg_rules.txt](/schedule/reference/#fare_leg_rulestxt)|`fare_product_id`|
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following tables presents a simple fare product (single ride $2.75 USD). 
+    </p>
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/schedule/reference/#fare_productstxt"><b>fare_products.txt</b></a> <br>
+        </p>
+
+        | fare_product_id  | fare_product_name  | amount  | currency  |
+        |------------------|--------------------|---|---|
+        | single_ride | Single Ride Fare |  2.75 | USD  |
+
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/schedule/reference/#fare_leg_rulestxt"><b>fare_leg_rules.txt</b></a> <br>
+        </p>
+
+        | fare_product_id  |
+        |------------------|
+        | single_ride |
+
+
+## Fare Media
+
+Fare Media defines the supported media that can be used to hold and/or validate a fare product. This refers to physical or virtual containers such as a paper ticket, a rechargeable transit card or even contactless payment with credit cards or smartphones.
+
+| Files included                   | Fields included   |
+|----------------------------------|-------------------|
+|[fare_media.txt](/schedule/reference/#fare_mediatxt)|`fare_media_id`, `fare_media_name`, `fare_media_type`|
+|[fare_products.txt](/schedule/reference/#fare_productstxt)|`fare_media_id`|
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table shows a snippet of different fare media in the San Francisco Bay Area. `Clipper` is described as a physical transit card with `fare_media_type=2`. `SFMTA Munimobile` is described as a mobile app with `fare_media_type=2`. `Cash` which is given directly to the driver without a ticket, is `fare_media_type=0`.
+    </p>
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/schedule/reference/#fare_mediatxt"><b>fare_media.txt</b></a> <br>
+        </p>
+
+        | fare_media_id | fare_media_name  | fare_media_type |
+        |---------------|------------------|-----------------|
+        | clipper       | Clipper          | 2               |
+        | munimobile    | SFMTA MuniMobile | 4               |
+        | cash          | Cash             | 0               |  
+
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/schedule/reference/#fare_productstxt"><b>fare_products.txt</b></a> <br>
+        </p>
+
+        | fare_product_id  |
```

Comment:
Probably need to include media here

Reply:
Included all required fields for fare_products.txt in addition to fare_media